File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/DetalleMenu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace DDS_Restaurant_Solution.Models
{
    public class DetalleMenu
    {
        [Key]
        public int idDetalleMenu { get; set; }

        public int cantidad { get; set; }

        public int idMenu { get; set; }
        [ForeignKey("idMenu")]
        public Menu Menu { get; set; }

        public int idProducto { get; set; }
        [ForeignKey("idProducto")]
        public Producto Productos { get; set; }


    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/DetalleMenu.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class DetalleMenu
    <Key>
    Public Property idDetalleMenu As Integer
    Public Property cantidad As Integer
    Public Property idMenu As Integer
    <ForeignKey("idMenu")>
    Public Property Menu As Menu
    Public Property idProducto As Integer
    <ForeignKey("idProducto")>
    Public Property Productos As Producto
End Class
